### PR TITLE
Updated commenting for element descritpion and parameter descriptions

### PR DESCRIPTION
**Pull request by jarpinosynthace**
Updated commenting such that the autogenerated info about parameters from within the UI has description for each parameter and what they do. Also to help guide the user of how to use the element. Actual steps in code has not had any comments updated but this may be necessary at a later stage of documentation. These autogenerated documentation pages will be generated from these comments so need to be as descriptive as possible moving forwards.

### CODE REVIEW

File: starter/Aliquot/Aliquot.an [hunk 1]
```diff
@@ -27,14 +30,14 @@ Data (
 
 // Physical Inputs to this protocol with types
 Inputs (
-	Solution *wtype.LHComponent 
-	OutPlate *wtype.LHPlate // this time we're specifying what plate we're using
+	Solution *wtype.LHComponent // This Physical input will have associated properties to determine how the liquid should be handled, e.g. is your Solution water or is it Glycerol. If your physical liquid does not exist in the Antha LHComponent library then create a new one on the fly with the NewLHComponents element and wire the output into this input. Alternatively wire a solution made by another element into this input to be alliquoted.
```
Comment (chr1sgrant) [inline starter/Aliquot/Aliquot.an]:
```
If your physical liquid does not exist in the Antha LHComponent library then create a new one on the fly with the NewLHComponents element 
```
should be AddNewLHComponent element

File: lib/Aliquot_.go [hunk 2]
```diff
@@ -1,5 +1,8 @@
-// Aliquot a solution into a specified plate.
-// optionally premix the solution before aliquoting
+// The Aliquot element will transfer a user defined number of aliquots of a specified volume of a solution into a specified plate type.
+// The user has the option to premix the solution to be aliquoted if components sink in the solution. Upstream elements that produce
```
Comment (chr1sgrant) [inline lib/Aliquot_.go]:
```
// The user has the option to premix the solution to be aliquoted if the input solution tends to sediment or separate when left to stand (e.g. a suspension of cells in media) or has recently been thawed. 
```

File: starter/Aliquot/Aliquot.an [hunk 3]
```diff
@@ -11,12 +14,12 @@ import (
 
 // Input parameters for this protocol (data)
 Parameters (
-	SolutionVolume Volume
-	VolumePerAliquot  Volume
-	NumberofAliquots int
-	PreMix bool // optional field. Select if the solution to be aliquoted should be premixed prior to transer
-	ChangeSolutionName string // optional field to change the name of the component
-	OptimisePlateUsage bool
+	SolutionVolume Volume // This parameter represents the volume of solution that you have in the lab available to be aliquoted. It does not represent the total volume to be aliquoted or the volume of liquid that will be used.
```
Comment (chr1sgrant) [inline starter/Aliquot/Aliquot.an]: I wonder if we want to make this an optional parameter.

Comment (jarpinosynthace) [inline starter/Aliquot/Aliquot.an]: Its up to you, i think it is beneficial, especially when wiring in from an upstream element, but when you are using it as a stand alone element it is a bit ambiguous

Comment (jarpinosynthace) [inline starter/Aliquot/Aliquot.an]: I think for now we can leave it as it is as i think it is more rigorous to have these checks in place to make sure people aren't doing something stupid

**Pull request merged**